client/joinus_client/src/components/Events/EventCard.js

```diff
@@ -63,16 +114,112 @@ export default function EventCard(props) {
             {shortenText(description)}
           </Typography>
         </CardContent>
-        <CardActions>
-          <Button onClick={submitHandler} size="small">
-            Learn More
-          </Button>
-          <Button onClick={() => {}} size="small">
-            Join Event
-          </Button>
-          <AttendeeNumDisplay attendeelist={attendeelist} />
-        </CardActions>
+        {user.id === owner_id ? 
+                <CardActions>
+                <Button onClick={submitHandler} size="small">
+                  Learn More
+                </Button>
+                <Button onClick={(e) => setOpen(true)} size="small">
+                  Edit Event
+                </Button>
+                <AttendeeNumDisplay attendeelist={attendeelist} />
+              </CardActions>
+      : <CardActions>
+      <Button onClick={submitHandler} size="small">
+        Learn More
+      </Button>
+      <Button onClick={() => {}} size="small">
+        Join Event
+      </Button>
+      <AttendeeNumDisplay attendeelist={attendeelist} />
+    </CardActions> }
       </Card>
+      <StyledModal
```

Comment:
HI Kyler! - Can you please change StyledModal to Modal please? -- <StyledModal> causes an issue with retreiving/sending data to the backend (it causes the modal to reload after every input event)